azurelinuxagent/ga/exthandlers.py

```diff
@@ -947,6 +973,17 @@ def collect_ext_status(self, ext):
 
         return ext_status
     
+    def is_ext_status_success(self, ext):
+        status = self.collect_ext_status(ext)
+        if status is None:
+            return (True, status)
+        if status.status != "success":
```

Comment:
Can you really rely on this value, and this casing?

Reply:
Agent throws exception when the status data is not present or the value is invalid. So I think we can rely on this value and casing.